### DISCUSSION TITLE
Receiving a bunch of warnings for previously working action

### DISCUSSION THREAD

**Discussion by Andre601** [lowlighter/metrics#1222] [Q&A]
After updating to 3.28 from 3.27 do SVGs now display the same warning over and over again:
"repositories.batch" has no effect because "repositories: true" is not set "

![image](https://user-images.githubusercontent.com/11576465/189237249-4cc5eb5c-2e1c-4c3b-a495-b06a5cd02d90.png)

The action worked fine before and from what I can tell was there never any warning like this...
What is wrong?

**Comment by lowlighter** [comment]
Seems to be a regression due to last release, will try to provide a fix in the coming days, sorry for this 😅

**Reply by Andre601** [reply]
Any updates on this?

**Reply by lowlighter** [reply]
Should be fixed once #1244 lands

**Comment by Andre601** [comment]
Issue is still present with v3.29
![grafik](https://user-images.githubusercontent.com/11576465/202901626-faf1651b-f7a2-426e-bf95-2be6189f5280.png)

I also checked the SVGs themself in case it was my browser cache but those are also the same.

**Reply by Andre601** [reply]
Also, to add to this are the options confusing... It f.e. mentions discussions.categories having no effect because discussions: true isn't set, but I do use an embed option with discussions: true and categories set to false, so it should work. Discussions are enabled within the workflow file too.

**Reply by lowlighter** [reply]
Sorry you still have this issue 😕 

I'll reopen #1229 

In the meantime, you should be able to get ride of these by adding this to your workflow:
```yml
extras_css: |
  .field.warning { display: none; }
```

**Reply by Andre601** [reply]
That hides it. Thanks.
Also, from what I could see are the warnings different this time. Before was it always the same warning but now is it different for each image/embed.
